apps/src/code-studio/pd/foorm/library_editor/FoormLibraryEditorHeader.jsx

```diff
@@ -42,16 +39,15 @@ const PREVIEW_ON = 'preview-on';
 const PREVIEW_OFF = 'preview-off';
 const TIME_FORMAT = 'h:mm a';
 
-class FoormEditorHeader extends Component {
+class FoormLibraryEditorHeader extends Component {
```

Comment:
This one seems like it could be combined pretty easily - I'm not sure the best way to handle getting the values from redux, but one option would be for this component to accept all the props as optional, with a comment about when they are each present.